### ISSUE TITLE
Does not support javascript projects

### ISSUE THREAD

**Issue by adborden** [goodeggs/unionized#18]
Hiya folks,

Hope you're all well. I'm running a babel (es6) project but [goodeggs/goodeggs-json-schema-validator](https://github.com/goodeggs/goodeggs-json-schema-validator) doesn't compile it's coffeescript, so when using unionized, I get this issue:

```
/Users/aborden/projects/search-api/node_modules/unionized/node_modules/goodeggs-json-schema-validator/validator/index.coffee:1
nction (exports, require, module, __filename, __dirname) { tv4 = require 'tv4'
                                                                         ^^^^^
SyntaxError: Unexpected string
    at Module._compile (module.js:439:25)
    at Module._extensions..js (module.js:474:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/aborden/projects/search-api/node_modules/babel/node_modules/babel-core/lib/api/register/node.js:214:7)
    at Object.Module._extensions.(anonymous function) [as .js] (/Users/aborden/projects/search-api/node_modules/istanbul/lib/hook.js:109:37)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/aborden/projects/search-api/node_modules/unionized/lib/json_schema_factory.js:15:13)
    at Module._compile (module.js:456:26)
    at Module._extensions..js (module.js:474:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/aborden/projects/search-api/node_modules/babel/node_modules/babel-core/lib/api/register/node.js:214:7)
    at Object.Module._extensions.(anonymous function) [as .js] (/Users/aborden/projects/search-api/node_modules/istanbul/lib/hook.js:109:37)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

**Comment by demands** [comment]
Ah thanks for the pointer. Looks like this'll get fixed if we implement #19.

**Comment by demands** [comment]
Or we could merge in the new version of goodeggs-json-schema-validator -- that seems like a great option too!
